app/lib/cfe_connection.rb

```diff
@@ -93,6 +93,23 @@ def create_regular_payments(assessment_id, income_form, outgoings_form)
     create_record(assessment_id, "regular_transactions", regular_transactions:) if regular_transactions.any?
   end
 
+  def create_benefits(assessment_id, benefits)
+    state_benefits = benefits.map do |benefit|
+      payments = [2, 1, 0].map do |v|
```

Comment:
I'll do a full review tomorrow, but one thing to flag up immediately: CFE won't believe that your payments are fortnightly or weekly if you only provide 3 instances of them. [It thinks](https://github.com/ministryofjustice/check-financial-eligibility/blob/main/app/services/utilities/regular_period_analyser.rb) that you are providing 3 months' worth of benefits payments, so if they're weekly there should be 12 or 13 of them, and if they are fortnightly there must be 6 or 7 of them. See the logic in EmploymentHandler (added as a bugfix for EL-402)